src/components/duelling-picklist/duelling-picklist.stories.mdx

```diff
@@ -6,17 +6,11 @@ import PicklistDivider from "./picklist-divider/picklist-divider.component";
 import PicklistPlaceholder from "./picklist-placeholder/picklist-placeholder.component";
 import DuellingPicklist from "./duelling-picklist.component";
 import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
-import {
-  Default,
-  AlternativeSearch,
-  Grouped,
-  InDialog,
-} from "./duelling-picklist-test.stories";
 
-<Meta
-  title="DuellingPicklist"
-  parameters={{ info: { disable: true }, chromatic: { disable: true } }}
-/>
+import { Default } from "./duelling-picklist-test.stories";
+import * as stories from "./duelling-picklist.stories";
+
+<Meta title="DuellingPicklist" parameters={{ info: { disable: true } }} />
```

Comment:
I noticed we are now making all the stories (except `InDialog` and `CustomTooltipMessage`) visible to chromatic - I can imagine this was intentional but just wanted to mention it in case?

Reply:
Yes, that was intentional and discussed with Igor